development/video-recorder-app/pages/video-recorder/index.mdx

```diff
@@ -0,0 +1,164 @@
+| `autoStop`             | `boolean`                         | `true`                   | Automatically stops recording after a specified time.                                                                                                                                        |
+| `autoPreview`          | `boolean`                         | `true`                   | Automatically shows a preview.                                                                                                |
+| `timeInMs`             | `number`                          | `15000`                  | Time in milliseconds for recording (only applicable if `autoStop` is true).                                                    |
+| `handleStateChange`    | `(state: VideoState) => void`     |                          | Callback function triggered on state change.                                                                                                                                                |
+| `customhandleDownload` | `(file: any) => void`              |                          | Custom function for handling the download of the recorded file.                                                                                                                             |
+| `customCountDown`      | `(count: number) => React.ReactNode` |                      | Custom component or function for displaying the countdown during recording.                                                                                                                |
+| `customRetakeButton`   | `(onRetake: any) => React.ReactNode` |                     | Custom component or function for the retake button.                                                                                                                                         |
+| `customStopRecording`  | `(onStop: any) => React.ReactNode`  |                         | Custom component or function for the stop recording button.                                                                                                                                |
+| `customStartRecording` | `(onStart: any) => React.ReactNode` |                        | Custom component or function for the start recording button.                                                                                                                               |
+| `customDownloadButton` | `(onDownload: any) => React.ReactNode` |                    | Custom component or function for the download button.                                                                                                                                      |
+| `mediaConstraints`     | `MediaStreamConstraints`           |                          | Media constraints to customize video and audio settings.                                                                                                                                   |
```

Comment:
Can we have for description the generally used keys for this, like video and audio etc. As those are the most necessary keys that will be used by the user.